shell-pipelines/sort-uniq-head-tail/script-07.sh

```diff
@@ -6,3 +6,4 @@ set -euo pipefail
 # TODO: Write a command to show how many times anyone has entered and exited.
 # It should be clear from your script's output that there have been 5 Entry events and 4 Exit events.
 # The word "Event" should not appear in your script's output.
+tail -9 events-with-timestamps.txt | sort -k3 | awk '{print $3}' | uniq -c
```

Comment:
This works, but I probably would've started it with `head -n+2` rather than `tail -9` - can you think about why that is? What's the difference between the two?